src/test/java/uk/gov/register/db/mappers/EntryMapperTest.java

```diff
@@ -15,22 +14,21 @@
 import static org.junit.Assert.assertThat;
 
 public class EntryMapperTest {
-    @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    @Rule
+    public WipeDatabaseRule wipeDatabaseRule = new WipeDatabaseRule("address");
 
     @Test
     public void map_returnsSameDateAndTimeInUTC() throws Exception {
         String expected = "2016-07-15T10:00:00Z";
         Instant expectedInstant = Instant.parse(expected);
 
-        DBI dbi = new DBI("jdbc:postgresql://localhost:5432/ft_openregister_java?user=postgres&ApplicationName=EntryMapperTest");
-        Handle h = dbi.open();
-        EntryQueryDAO entryQueryDAO = dbi.onDemand(EntryQueryDAO.class);
+        DBI dbi = new DBI("jdbc:postgresql://localhost:5432/ft_openregister_java_address?user=postgres&ApplicationName=EntryMapperTest");
 
-        h.execute("insert into entry(entry_number, sha256hex, timestamp) values(5, 'abcdef', :timestamp)",expectedInstant.getEpochSecond());
-
-        // this method implicitly invokes EntryMapper
-        Collection<Entry> allEntriesNoPagination = entryQueryDAO.getAllEntriesNoPagination();
+        Collection<Entry> allEntriesNoPagination = dbi.withHandle(h -> {
```

Comment:
actually, it doesn't. `dbi.withHandle` just makes a `Handle` (which wraps a `Connection`) available.  It doesn't imply any sort of transactional operation - that needs `dbi.{in,use}Transaction`.

(I had to check this locally before replying though)